ci.jenkins.io.tf

```diff
@@ -177,8 +189,12 @@ resource "azuread_application_password" "ci_jenkins_io" {
 }
 
 # Allow application to manage AzureRM resources inside the agents resource groups
-resource "azurerm_role_assignment" "ci_jenkins_io_contributor_in_agent_resourcegroup" {
-  scope                = "${data.azurerm_subscription.jenkins.id}/resourceGroups/${azurerm_resource_group.ci_jenkins_io_agents.name}"
+moved {
+  from = azurerm_role_assignment.ci_jenkins_io_contributor_in_agent_resourcegroup
+  to   = azurerm_role_assignment.ci_jenkins_io_contributor_in_eastus_agent_resourcegroup
+}
+resource "azurerm_role_assignment" "ci_jenkins_io_contributor_in_eastus_agent_resourcegroup" {
+  scope                = "${data.azurerm_subscription.jenkins.id}/resourceGroups/${azurerm_resource_group.eastus_ci_jenkins_io_agents.name}"
```

Comment:
```suggestion
  scope                = azurerm_resource_group.eastus_ci_jenkins_io_agents.id
```

Reply:
i use this a lot, it's the right syntax 😄

Reply:
the docs do this although only show subscription level: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/role_assignment#example-usage-custom-role--user

Reply:
I'm not forgetting about this but I'll do on a subsequent PR (I would really want to use the `id` : way simpler)

Reply:
https://github.com/jenkins-infra/azure/pull/414